server/controllers/stock/reports/stock_expiration_report.handlebars

```diff
@@ -59,32 +59,32 @@
     <table class="table table-condensed table-bordered table-report">
       <thead>
         <tr>
-          <th>{{translate 'STOCK.INVENTORY'}}</th>
-          <th>{{translate 'STOCK.LOT'}}</th>
-          <th>{{translate 'STOCK.EXPIRATION'}}</th>
-          <th>{{translate 'STOCK.CMM'}}</th>
-          <th>{{translate 'STOCK.CURRENT_QUANTITY'}}</th>
-          <th>{{translate 'STOCK.RISK_QUANTITY'}}</th>
-          <th>{{translate 'TABLE.COLUMNS.UNIT'}}</th>
-          <th>{{translate 'REPORT.STOCK.UNIT_COST'}}</th>
-          <th>{{translate 'STOCK.RISK_VALUE'}}</th>
-          <th>{{translate 'STOCK.STATUS.LABEL'}}</th>
+          <th width="34%">{{translate 'STOCK.INVENTORY'}}</th>
+          <th width="12%">{{translate 'STOCK.LOT'}}</th>
+          <th width="8%">{{translate 'STOCK.EXPIRATION'}}</th>
+          <th width="6%">{{translate 'STOCK.CMM'}}</th>
+          <th width="6%">{{translate 'STOCK.CURRENT_QUANTITY'}}</th>
+          <th width="6%">{{translate 'STOCK.RISK_QUANTITY'}}</th>
+          <th width="6%">{{translate 'TABLE.COLUMNS.UNIT'}}</th>
+          <th width="5%">{{translate 'REPORT.STOCK.UNIT_COST'}}</th>
+          <th width="5%">{{translate 'STOCK.RISK_VALUE'}}</th>
+          <th width="12%">{{translate 'STOCK.STATUS.LABEL'}}</th>
         </tr>
       </thead>
 
       <tbody>
         {{#each depot.rows as |lot| }}
           <tr>
-            <td>{{lot.text}}</td>
-            <td>{{lot.label}}</td>
-            <td>{{date lot.expiration_date}}</td>
-            <td class="text-right">{{lot.avg_consumption}}</td>
-            <td class="text-right">{{lot.quantity}}</td>
-            <td class="text-right">{{lot.quantity_at_risk}}</td>
-            <td>{{lot.unit_type}}</td>
-            <td>{{currency lot.unit_cost ../../metadata.enterprise.currency_id}}</td>
-            <td class="text-right">{{currency lot.value ../../metadata.enterprise.currency_id}}</td>
-            <td class="{{lot.classKey}}">{{translate lot.statusKey}}</td>
+            <td width="34%">{{lot.text}}</td>
+            <td width="12%">{{lot.label}}</td>
+            <td width="8%">{{date lot.expiration_date}}</td>
+            <td width="6%" class="text-right">{{lot.avg_consumption}}</td>
+            <td width="6%" class="text-right">{{lot.quantity}}</td>
+            <td width="6%" class="text-right">{{lot.quantity_at_risk}}</td>
+            <td width="6%">{{lot.unit_type}}</td>
+            <td width="5%">{{currency lot.unit_cost ../../metadata.enterprise.currency_id}}</td>
```

Comment:
Can you make this cost `class="text-right"` as well?

Reply:
Done